README.md

```diff
@@ -1 +1,105 @@
-# Capstone-Pro-01
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Content's
+
+- [📗 Table of Content's](#-table-of-contents)
+- [Project Name](#project-name)
+  - [Built With](#built-with)
+    - [Key Features ](#key-features-)
+  - [Live Demo (if available)](#live-demo-if-available)
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing](#-contributing)
+  - [Show your support](#show-your-support)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License](#-license)
+
+# Project Name
+- AHMADI GYM Project
+
+> A website for a GYM having multiple pages.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Bootstrap
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+I find the following features to be intresting:
+
+- **The Layout of Website**
+- **Flexbox and grid for structuring instead of floats**
+
+## Live Demo (if available)
+
+[Live Demo Link](https://sulaimanahmadi.github.io/Capstone-Pro-01/)
+
+
+## Getting Started
+
+You can get started by cloning the project or vist the demo link
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Have Git and GitHub
+### Setup
+Copy HTTPS link of the repo and clone it in your local machine.
+### Install
+Have a code editor to make changes.
+### Usage
+This project for a gym that any one can use it.
+### Run tests
+Run the linter before pushing the project.
+### Deployment
+This is already deployet just checkout the demo link
+
+
+## Authors
+
+👤 **Sulaiman Ahmadi**
+
+- GitHub: [@sulaimanahmadi](https://github.com/sulaimanahmadi)
+- Twitter: [@Sulaiman Ahamdi](https://twitter.com/sahmadi4all)
+- LinkedIn: [@Sulaiman Ahmadi](https://www.linkedin.com/in/mohammad-sulaiman-ahmadi-555315261/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+In the future, I will Add 
+
+- [ ] **Featured section**
+- [ ] **About section**
+- [ ] **Desktop Version**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://sulaimanahmadi.github.io/Capstone-Pro-01/)
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] I would kindly suggest you acknowledge the original design owner  (Cindy Shin) as requested in the project requirement in the `Acknowledgement` heading of the Readme file.